src/script.js

```diff
@@ -197,6 +197,17 @@
 			document.body.appendChild(a);
 			a.click();
 			a.remove();
+		};
+
+		saveHtmlBtn.addEventListener('click', function (e) {
+			save_file();
+		});
+		
+		window.addEventListener('keydown',function(event) {
+			if ((event.ctrlKey)&&(event.keyCode==83)){
```

Comment:
Mention a comment that this is code for `Ctrl + S` as 83 might not be recognizable to every developer.